ucan-invocation/types.ts

```diff
@@ -48,16 +62,17 @@ export type UcanInvocationType = 'workflow' | 'receipt'
 
 export interface UcanInvocation {
   carCid: string
+  invocationCid: string
```

Comment:
in https://github.com/web3-storage/w3infra/blob/9897890c65968f7d47a4b3b520f736863343642b/upload-api/ucan-invocation.js#L169

Reply:
Yes but, only for receipts.

I ended up re-typing these definitions in `billing` to more accurately reflect what's there and not get errors for properties that don't always exist:

https://github.com/web3-storage/w3infra/blob/9897890c65968f7d47a4b3b520f736863343642b/billing/lib/api.ts#L210-L240